b2g/installer/package-manifest.in

```diff
@@ -972,3 +972,8 @@ bin/libfreebl_32int64_3.so
 
 @RESPATH@/components/simpleServices.js
 @RESPATH@/components/utils.manifest
+
+#ifdef MOZ_B2G_REMOTECONTROLSERVICE
+[remotecontrol]
+@RESPATH@/res/remotecontrol/*
```

Comment:
Putting JS file in `/res` looks weird to me.

Here is my suggestion if there is only one such JS file for remote control:

- move `b2g/remotecontrol/command.js` to `b2g/chrome/content/remotecontrol.js`
- add `content/remotecontrol.js                     (content/remotecontrol.js)` in `b2g/chrome/jar.mn` with compile option wrapped.
- use `chrome://b2g/content/remotecontrol.js` as the URL for loading this JS file.